src/core/TChem_Driver.cpp

```diff
@@ -48,9 +49,11 @@ void initialize(const char* chemFile, const char* aeroFile, const char* numerics
 
   using ordinal_type = TChem::ordinal_type;
 
+//  ordinal_type nBatch = 1;
```

Comment:
Should this comment be deleted?

Reply:
This comment is outdated and was removed in 42a772624936477c979abcfe3345687edeb32e66